d2l-search-facets-grouping.html

```diff
@@ -0,0 +1,135 @@
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../d2l-inputs/d2l-input-checkbox-spacer.html">
+<link rel="import" href="../d2l-typography/d2l-typography-shared-styles.html">
+
+<dom-module id="d2l-search-facets-grouping">
+	<template strip-whitespace>
+		<style>
+			:host {
+				display: flex;
+				flex-direction: column;
+			}
+			.d2l-search-facets-grouping-label {
+				@apply --d2l-body-compact-text;
+				font-weight: bold;
+			}
+
+			.d2l-search-facets-grouping-show-more > button {
+				@apply --d2l-body-compact-text;
+				border: none;
+				color: var(--d2l-color-celestine);
+				font-family: inherit;
+				outline: none;
+				padding: 0px;
+			}
+
+			.d2l-search-facets-grouping-show-more > button:hover,
+			.d2l-search-facets-grouping-show-more > button:focus {
+				cursor: pointer;
+				font-weight: bold;
+			}
+
+		</style>
+		<span class="d2l-search-facets-grouping-label">[[label]]</span>
```

Comment:
Seems like a good opportunity to use `<fieldset>` and `<legend>` to group the checkboxes semantically together. To a screen reader, it makes things super helpful, as when they visit a checkbox it will announce both the checkbox label (e.g. "New Courses only") but also the group legend text (e.g. "My Status").